lib/transformations/utils.test.js

```diff
@@ -157,4 +157,11 @@ var a = { plugs: [] }
 			.filter(p => utils.findObjWithOneOfKeys(p, ['a'])).size()).toEqual(1);
 		});
 	});
+
+	describe('getRequire', () => {
+		it('should create a require statement', () => {
+			const require = utils.getRequire(j, 'filesys', 'fs');
+			expect(require).toMatchSnapshot();
```

Comment:
This should probably create snapshot for the source code. Right now it's for AST.

`expect(require.toSource()).toMatchSnapshot()`?